.gitignore

```diff
@@ -118,3 +118,9 @@ env/
 
 ### Configuration files ###
 .env
+
+### NPM things ###
+node_modules/
+
+### Tailwin Generated CSS ###
+todo_app/static/css/index.css
```

Comment:
I agree with this for the record, if your css file is generated then I expect it to be `.gitignored` by default (as with JS files that gets compiled from TS)